### ISSUE TITLE
Mint a new release? When?

### ISSUE THREAD

**Issue by magnars** [clojure-emacs/clj-refactor.el#78]
Hi guys! Thanks for all of your hard work on clj-refactor and the nrepl middleware this summer and fall. :heart: I hope I can contribute more in 2015 than I did in 2014. In the meantime, would you like me to mint a new release of clj-refactor.el? When?

**Comment by benedekfazekas** [comment]
find usages/rename symbol for locals done. acc to my knowledge last outstanding issue is macro handling...

**Comment by expez** [comment]
Great job on adding support for locals!  As far as macros go, I propose we do the following:

1. Check if macro is defined in the project using `cider-var-info`
2. Search all files for `$macro`
3. For each file and each occurrence, check if `$macro` resolves to the same thing as found in step 1 and if so return the entry

This is likely to be pretty god damn slow for large projects, but we can add caching just like we did with ASTs.  Renaming macros is a less common use-case--I think--so I'm fine with adding caching at some later time.

**Comment by magnars** [comment]
This is getting long, so I opened up a new issue: #105 - seems like there is more to do apart from macros.

**Comment by benedekfazekas** [comment]
so guys, how are we with 1.0? I know/guess we are all quite busy but it feels 1.0 is soooo very close ;) 

a few points/questions:
- I am looking at macro handling atm: don't see an easy win so far... can we possibly do 1.0 without it or that would be a disappointment?
- any outstanding problems you see blocking 1.0?
- what do you think is missing and should be defo in 1.0 scope?
- I think we should sync clj-refactor and refactor-nrepl versions for 1.0 and from that point on

**Comment by expez** [comment]
I outlined a strategy for handling macros above that I think should be fairly easy to implement but not very performant.  I don't really care if this is included or not myself.  Macros make up a very small percentage of the code I write so renaming them the old fashion way is perfectly fine.

I'd like to fix the open issues in refactor nrepl I've assigned to myself.  Both are trivial to fix, I just haven't had the time lately.  I'll get that done in the next days.

It would be great to fix the classloader isolation issue with a post-delegation classloader, but I don't think I'll have time to dive into that.  I'm OK with leaving that out if nobody else has time either.

I've written a far bit of clojure in 2015 and I quite enjoy clj-refactor!  There's a ton of features in here now!  I had forgotten all about `extract-function` and was delighted to rediscover it the other day :)
